fonts.mk

```diff
@@ -0,0 +1,521 @@
+
+	cd $(BUILD_WORK)/font-schumacher-misc && autoreconf -fiv && ./configure -C \
+		--build=$$($(BUILD_MISC)/config.guess) \
+		--host=$(GNU_HOST_TRIPLE) \
+		--prefix=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX)
+	+$(MAKE) -C $(BUILD_WORK)/font-schumacher-misc
+	+$(MAKE) -C $(BUILD_WORK)/font-schumacher-misc install \
+		DESTDIR=$(BUILD_STAGE)/fonts
+	+$(MAKE) -C $(BUILD_WORK)/font-schumacher-misc install \
+		DESTDIR=$(BUILD_BASE)
+	touch $(BUILD_WORK)/font-schumacher-misc/.build_complete
+
+	cd $(BUILD_WORK)/font-screen-cyrillic && autoreconf -fiv && ./configure -C \
+		--build=$$($(BUILD_MISC)/config.guess) \
+		--host=$(GNU_HOST_TRIPLE) \
+		--prefix=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX) 
+	+$(MAKE) -C $(BUILD_WORK)/font-screen-cyrillic
+	+$(MAKE) -C $(BUILD_WORK)/font-screen-cyrillic install \
+		DESTDIR=$(BUILD_STAGE)/fonts
+	+$(MAKE) -C $(BUILD_WORK)/font-screen-cyrillic install \
+		DESTDIR=$(BUILD_BASE)
+	touch $(BUILD_WORK)/font-screen-cyrillic/.build_complete
+
+	cd $(BUILD_WORK)/font-sony-misc && autoreconf -fiv && ./configure -C \
+		--build=$$($(BUILD_MISC)/config.guess) \
+		--host=$(GNU_HOST_TRIPLE) \
+		--prefix=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX) 
+	+$(MAKE) -C $(BUILD_WORK)/font-sony-misc
+	+$(MAKE) -C $(BUILD_WORK)/font-sony-misc install \
+		DESTDIR=$(BUILD_STAGE)/fonts
+	+$(MAKE) -C $(BUILD_WORK)/font-sony-misc install \
+		DESTDIR=$(BUILD_BASE)
+	touch $(BUILD_WORK)/font-sony-misc/.build_complete
+
+	cd $(BUILD_WORK)/font-sun-misc && autoreconf -fiv && ./configure -C \
+		--host=$(GNU_HOST_TRIPLE) \
+		--prefix=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX) 
+	+$(MAKE) -C $(BUILD_WORK)/font-sun-misc
+	+$(MAKE) -C $(BUILD_WORK)/font-sun-misc install \
+		DESTDIR=$(BUILD_STAGE)/fonts
+	+$(MAKE) -C $(BUILD_WORK)/font-sun-misc install \
+		DESTDIR=$(BUILD_BASE)
+	touch $(BUILD_WORK)/font-sun-misc/.build_complete
+
+	cd $(BUILD_WORK)/font-winitzki-cyrillic && autoreconf -fiv && ./configure -C \
+		--build=$$($(BUILD_MISC)/config.guess) \
+		--host=$(GNU_HOST_TRIPLE) \
+		--prefix=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX) 
+	+$(MAKE) -C $(BUILD_WORK)/font-winitzki-cyrillic
+	+$(MAKE) -C $(BUILD_WORK)/font-winitzki-cyrillic install \
+		DESTDIR=$(BUILD_STAGE)/fonts
+	+$(MAKE) -C $(BUILD_WORK)/font-winitzki-cyrillic install \
+		DESTDIR=$(BUILD_BASE)
+	touch $(BUILD_WORK)/font-winitzki-cyrillic/.build_complete
+
+	cd $(BUILD_WORK)/font-xfree86-type1 && autoreconf -fiv && ./configure -C \
+		--build=$$($(BUILD_MISC)/config.guess) \
+		--host=$(GNU_HOST_TRIPLE) \
+		--prefix=$(MEMO_PREFIX)$(MEMO_SUB_PREFIX) 
+	+$(MAKE) -C $(BUILD_WORK)/font-xfree86-type1
+	+$(MAKE) -C $(BUILD_WORK)/font-xfree86-type1 install \
+		DESTDIR=$(BUILD_STAGE)/fonts
+	+$(MAKE) -C $(BUILD_WORK)/font-xfree86-type1 install \
+		DESTDIR=$(BUILD_BASE)
+	touch $(BUILD_WORK)/font-xfree86-type1/.build_complete
+endif
+
+fonts-package: fonts-stage
+# fonts.mk Package Structure
+	rm -rf $(BUILD_DIST)/fonts
+	
+# fonts.mk Prep fonts
+	cp -a $(BUILD_STAGE)/fonts $(BUILD_DIST)
```

Comment:
The fonts should be split up into separate packages.

Reply:
I don’t have a problem in this case.

Reply:
Keep it as one.

Reply:
I think this is a case where splitting is necessarily, I don’t need 32 fonts unless I’m using libreoffice, so all these fonts would be bloat, besides, splitting it would be super easy to do, you wouldn’t even need to create 32 make files, or have repeat lines if you do it the same was as #520

Reply:
That’s a good point. Agreed.